booking/src/main/java/ru/org.dsr/domain/Item.java

```diff
@@ -0,0 +1,56 @@
+package ru.org.dsr.domain;
+
+public class Item {
+    private ItemID itemID;
```

Comment:
I don't like such composite keys, if we need to associate entry in DB with this key, need to normalize DB, decompose and move this data to separate table.